### DISCUSSION TITLE
现在新增接口后，点击右上角的预览，并没有看到swagger文档

### DISCUSSION THREAD

**Discussion by codecodify** [fireboomio/product-manual#81] [Q&A]
本地飞布版本：v0.3.0
操作系统：Mac M1
本地新增两个接口，并标记上线状态，但是点击右上角的预览（http://127.0.0.1:9123/#/workbench/rapi?t=1692516733038），并没有看到接口的文档信息，中间是空白的

**Comment by AnsonCode** [comment]
方便提供下截图吗？包括控制台报错。
或者加我微信 luxuncode

**Reply by codecodify** [reply]
![31692771640_.pic_hd.jpg](https://img1.imgtp.com/2023/08/23/PQ0aBlMC.jpg)

**Comment by AnsonCode** [comment]
问题已修复，推荐使用FireboomV2.0：https://docs.fireboom.io/kuai-su-ru-men/kuai-su-shang-shou#ben-di-an-zhuang
